talend-component-maven-plugin/pom.xml

```diff
@@ -59,7 +60,7 @@
     <dependency>
       <groupId>org.apache.maven.plugin-tools</groupId>
       <artifactId>maven-plugin-annotations</artifactId>
-      <version>${mvn-minimal.version}</version>
+      <version>${maven-plugin-annotations.version}</version>
```

Comment:
Where do we use "mvn-minimal.version"?

Reply:
It's used in requireMavenVersion for enforcer plugin.